test/utils.go

```diff
@@ -75,7 +84,8 @@ func setupProwConfig(resourceType string) string {
 	project := p.Name
 
 	go func(c *boskosclient.Client) {
-		for range time.Tick(time.Minute * 5) {
+		select {
```

Comment:
Can you explain what you are trying to do here?

Reply:
Here we are trying to update the freshness of this resource by doing another set.
The reason is Boskos has a [reaper](https://github.com/kubernetes-sigs/boskos/blob/a7ef97ee445d/README.md#other-components) component that cleans up stale resources, by periodically setting an expiration time(30min) on the resources that are in use(in "busy" state), and if if hasn't updated within the expiration time, it would set it to "dirty" state. And Janitor component cleans up all "dirty" resources". So to avoid resource in-use being accidentally cleaned up, we need to periodically update it.

Reply:
This needs a comment to describe this.

Reply:
Added the comment.

Reply:
You don't need to pass `boskos` it is already implicitly captured in the anonymous func

You should wrap this in a struct to keep things organized